src/Razor/src/Microsoft.VisualStudio.Editor.Razor/BackgroundDocumentGenerator.cs

```diff
@@ -265,12 +265,12 @@ private void ReportError(ProjectSnapshot project, Exception ex)
         {
             OnErrorBeingReported();
 
-            GC.KeepAlive(Task.Factory.StartNew(
+            _ = Task.Factory.StartNew(
```

Comment:
Were these also attempts to suppress CS0067?

Reply:
➡️ No, these were suppressing [VSTHRD110](https://github.com/Microsoft/vs-threading/blob/main/doc/analyzers/VSTHRD110.md)